deployability/modules/allocation/aws/provider.py

```diff
@@ -38,21 +39,25 @@ def _create_instance(cls, base_dir: Path, params: CreationPayload, config: AWSCo
         temp_dir = base_dir / temp_id
         credentials = AWSCredentials()
         if not config:
+            logger.debug(f"No config provided. Generating")
```

Comment:
The config could be provided as argument in the function or could be generated using the provided payload, I think it could be something like this:
```suggestion
            logger.debug(f"No config provided. Generating from payload")
```